test/Polly.Specs/Wrap/PolicyWrapSpecs.cs

```diff
@@ -308,7 +311,7 @@ public void Wrapping_nothing_using_static_wrap_strongly_typed_syntax_should_thro
     public void Wrapping_only_one_policy_using_static_wrap_strongly_typed_syntax_should_throw()
     {
         Policy<int> singlePolicy = Policy<int>.Handle<Exception>().Retry();
-        Action config = () => Policy.Wrap<int>(new[] { singlePolicy });
+        Action config = () => Policy.Wrap<int>(singlePolicy);
```

Comment:
Was this using an array to call a specific overload?

Reply:
<img width="767" alt="image" src="https://github.com/user-attachments/assets/29217def-e03b-4559-96ef-429f71b675ca">